resources/schema/spidacalc/calc/structure.schema

```diff
@@ -794,6 +794,9 @@
           "attachmentHeight": {
             "$ref": "../../general/measurable.schema"
           },
+          "height": {
```

Comment:
either height should be required by the schema or we should be able to handle json that does not include the push brace height

Reply:
does this mean that we don't want to run PushBraceHeightChangeSet.applyToStructure?

Reply:
we won't need PushBraceHeightChangeSet.applyToResults and applyToStructure anymore right?

Reply:
Right.